<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configwidget</class>
 <widget class="QWidget" name="configwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>335</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Main</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Advanced</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>System tray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shortcuts</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_6">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Default save directory</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="editDir">
              <property name="toolTip">
               <string>Path to default selection dir for saving</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonBrowse">
              <property name="minimumSize">
               <size>
                <width>112</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Browse filesystem</string>
              </property>
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_1">
           <property name="title">
            <string>Default file</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="labFilename">
                  <property name="text">
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="editFileName">
                  <property name="toolTip">
                   <string>Default filename</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="labFormat">
                  <property name="text">
                   <string>Format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbxFormat">
                  <property name="minimumSize">
                   <size>
                    <width>112</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Default saving image format</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="labCopyFileName">
             <property name="text">
              <string>Copy file name to the clipboard when saving</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbxCopyFileName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Do not copy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Copy file name only</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Copy full file path</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupQuality">
           <property name="title">
            <string>Image quality</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSlider" name="slideImgQuality">
              <property name="toolTip">
               <string>Image quality (1 - small file, 100 - high quality)</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labImgQualityCurrent">
              <property name="text">
               <string notr="true">Current</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>298</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="leftMargin">
            <number>16</number>
           </property>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkIncDate">
           <property name="toolTip">
            <string>Inserting current date time into saved filename</string>
           </property>
           <property name="text">
            <string>Insert current date and time in file name</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="labMask">
             <property name="text">
              <string>Template: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editDateTmeTpl">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>256</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labMaskExample">
           <property name="text">
            <string>Example: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkAutoSave">
           <property name="toolTip">
            <string>Automatically saving screenshots in grabbing process</string>
           </property>
           <property name="text">
            <string>Autosave screenshot</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <property name="leftMargin">
            <number>16</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkAutoSaveFirst">
             <property name="text">
              <string>Save first screenshot</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkAllowCopies">
           <property name="toolTip">
            <string>Allow run multiplies copy of ScreenGrab</string>
           </property>
           <property name="text">
            <string>Allow multiple instances of ScreenGrab</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbxEnableExtView">
           <property name="toolTip">
            <string>Open in external viewer on double click</string>
           </property>
           <property name="text">
            <string>Enable external viewer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkFitInside">
           <property name="toolTip">
            <string>If this is unchecked, the selected area
might become larger to fit to outer edges</string>
           </property>
           <property name="text">
            <string>Fit to edges inside selected area by right clicking</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>66</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkShowTray">
             <property name="text">
              <string>Show ScreenGrab in the system tray</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="labTrayMessages">
               <property name="text">
                <string>Tray messages:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbxTrayMsg">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Tray messages display mode</string>
               </property>
               <item>
                <property name="text">
                 <string>Never</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Tray mode</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Always</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="labTimeTrayMess">
               <property name="text">
                <string>Time of display tray messages</string>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="timeTrayMess">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Time to display tray messages</string>
               </property>
               <property name="suffix">
                <string> sec</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="checkInTray">
             <property name="toolTip">
              <string>Minimize to tray on click close button</string>
             </property>
             <property name="text">
              <string>Minimize to tray when closing</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_5">
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="treeKeys">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <column>
            <property name="text">
             <string>Action</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Shortcut</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Global shortcuts</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <item>
             <property name="text">
              <string>Full screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Active window</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Area select</string>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>Local shortcuts</string>
            </property>
            <item>
             <property name="text">
              <string>New screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Save screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Copy screen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Options</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Help</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Quit</string>
             </property>
            </item>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="labUsedShortcut">
             <property name="text">
              <string>Selected shortcut:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QKeySequenceWidget" name="keyWidget" native="true">
             <property name="noneText" stdset="0">
              <string>Not defined</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_1">
       <property name="standardButtons">
        <set>QDialogButtonBox::RestoreDefaults</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox_2">
         <property name="standardButtons">
          <set>QDialogButtonBox::Save</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox_3">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>qkeysequencewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
